#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = 'jsk_maps'


gen = ParameterGenerator()

gen.add('center_latitude', double_t, 0,
        'Latitude of center point', 35.70751, -90, 90)
gen.add('center_longitude', double_t, 0,
        'Longitude of center point', 139.76060, -180, 180)
gen.add('zoom_level', int_t, 0, 'Zoom level of map', 18, 1, 18)

exit(gen.generate(PACKAGE, 'static_map_publisher', 'StaticMapPublisher'))
